### DISCUSSION TITLE
I am getting an error and I need help fixing it

### DISCUSSION THREAD

**Discussion by Njasa2k** [oobabooga/text-generation-webui#870] [Q&A]
I'm following this tutorial and I get an error: https://github.com/oobabooga/text-generation-webui/wiki/LLaMA-model#4-bit-mode
```
mkdir repositories
cd repositories
git clone https://github.com/oobabooga/GPTQ-for-LLaMa.git -b cuda
cd GPTQ-for-LLaMa
python setup_cuda.py install
```
when I type in python setup_cuda.py install
I get this error: 
```
C:\Users\user\Downloads\oobabooga-windows\oobabooga-windows\text-generation-webui\repositories\GPTQ-for-LLaMa>python setup_cuda.py install
Traceback (most recent call last):
  File "C:\Users\user\Downloads\oobabooga-windows\oobabooga-windows\text-generation-webui\repositories\GPTQ-for-LLaMa\setup_cuda.py", line 2, in <module>
    from torch.utils import cpp_extension
  File "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\site-packages\torch\__init__.py", line 122, in <module>
    raise err
OSError: [WinError 127] The specified procedure could not be found. Error loading "C:\Users\user\AppData\Local\Programs\Python\Python310\lib\site-packages\torch\lib\c10_cuda.dll" or one of its dependencies.
```
Is there any way to fix this error?
It is obstructing me from fixing https://github.com/oobabooga/text-generation-webui/issues/824# 

**System Info**
`Windows, Nvidia RTX 2070 with Q-Max design, 8GB VRAM, 64GB RAM`

**Comment by WoefulWizard** [comment]
I am also getting this error.

**Comment by WoefulWizard** [comment]
Sorry I meant to add this.

![error](https://user-images.githubusercontent.com/130334903/230824968-21e6a64c-aa1a-4cad-906f-bfb0bdd5ffce.png)

**Comment by MaiJames** [comment]
Have you tried using the one-click installer?

I had the same problem before when I manually installed it.
I followed all the steps and it always failed setting up cuda so I could not run quantized models.

I deleted everyting and started all the setup again using the one-click installer and that did the trick for me.

**Reply by WoefulWizard** [reply]
For me, I am using the one-click install. I have not tried the manual yet. I wanted to see if there was a know fix for this first.
